app/js/app/services/payment.js

```diff
@@ -0,0 +1,63 @@
+angular.module('app.services')
```

Comment:
I need to be able to test the online payments for Stripe on the web app once this is in our staging environment.